tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "strictNullChecks": true,
+    "module": "none",
+    "target": "es6",
+    "rootDir": "./src"
+  },
```

Comment:
Add exclude section

Exemple of tsconfig (to inspire)
```json
{
  "compilerOptions": {
    "outDir": "dist",
    "module": "commonjs",
    "target": "es5",
    "declaration": true,
    "lib": [
      "dom",
      "es6",
      "scripthost"
    ],
    "sourceMap": true,
    "allowJs": false,
    "jsx": "react",
    "moduleResolution": "node",
    "rootDir": "src",
    "forceConsistentCasingInFileNames": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noImplicitAny": false,
    "strictNullChecks": true,
    "suppressImplicitAnyIndexErrors": true,
    "noUnusedLocals": true,
    "skipLibCheck": true
  },
  "exclude": [
    "node_modules",
    "build",
    "scripts",
    "acceptance-tests",
    "jest",
    "**/*.test.ts",
    "**/*.test.tsx"
  ],
  "include": [
    "src/**/*"
  ],
  "types": [
    "typePatches"
  ]
}
```

Reply:
done

Reply:
Missing important properties such as outDir etc...

Reply:
done